src/mjolnir/statistics.cc

```diff
@@ -26,6 +26,7 @@
 #include "midgard/aabb2.h"
 #include "midgard/logging.h"
 
+using namespace valhalla;
```

Comment:
did we need to add this namespace? if you didnt add it then you wouldnt have needed to qualify `RoadClass` was coming from `baldr`